File: API/Library_Data/Entities/Book.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Library_Data.Entities
{
    public class Book
    {
        public int Id { get; set; }
        public int BookCount { get; set; }

        public string? Title { get; set; }
        public string? Publisher { get; set; }
        public string? AuthorName { get; set; }
        public int TotalPages { get; set; }
        public string? Description { get; set; }
        public string? BookType { get; set; }
        public string? Category { get; set; }
        public string? Status { get; set; }    
        public string? ImagePath { get; set; }
        public string ISBN { get; set; }
    }
}

File: client/src/app/_models/book.ts
export class Book {
  id: number;
  bookCount: number;
  title: string;
  publisher: string;
  authorName: string;
  totalPages: number;
  description: string;
  imagePath: string;
  bookType: string;
  category: string;
  status: string;
  isbn: string;

  constructor() {
    this.id = 0;
    this.bookCount = 0;
    this.title = '';
    this.publisher = '';
    this.authorName = '';
    this.totalPages = 0;
    this.description = '';
    this.imagePath = '';
    this.bookType = '';
    this.category = '';
    this.status = '';
    this.isbn = '';
  }
}
